pkgs/development/libraries/libgweather/default.nix

```diff
@@ -22,13 +19,13 @@
 
 stdenv.mkDerivation rec {
   pname = "libgweather";
-  version = "40.0";
+  version = "3.91.0";
```

Comment:
Do we need to maintain both version? For example maybe evolution-data-server needs the old version? :thinking: 

- https://gitlab.gnome.org/GNOME/libgweather/-/commit/c4c161445d95c0a9e48b59f893988ef1f043e16a
- https://gitlab.gnome.org/GNOME/evolution-data-server/-/blob/3.43.2/CMakeLists.txt#L758

Reply:
I want to pull in patches where possible. E.g. https://gitlab.gnome.org/GNOME/evolution-data-server/-/merge_requests/90 looks reasonable.